MIGRATION.md

```diff
@@ -0,0 +1,253 @@
+# Migrating from `earthly` to EarthBuild
+
+[`earthly`](https://github.com/earthly/earthly) was originally developed by [Earthly
+Technologies](https://earthly.dev) as a corporately-sponsored open-source project.
+
+In 2025, earthly [announced a pivot to a different business
+model](https://web.archive.org/web/20250420142821/https://earthly.dev/blog/shutting-down-earthfiles-cloud/),
+no longer maintaining `earthly` to focus on entirely different products and directions.
+
+In response, the community has forked the project under the name `EarthBuild` to continue its development and maintenance.
+
+## What to Expect from EarthBuild
+
+<!--
+  TODO: It would be good to add more details here about the project's governance,
+  roadmap, and where to find community support (e.g., Slack, Discord, GitHub Discussions).
+  This is critical information for any organization considering this migration.
+-->
+
+EarthBuild is a community-driven project. This means development is no longer backed by a single corporation but by a collective of users and contributors.
+
+- **Stability**: The immediate goal of EarthBuild is to provide a secure, stable & reliable build tool for the community.
+- **Open Governance**: The project aims for an open and transparent governance model.
+- **Community Support**: Support is available through community channels.
+
+## Key Changes
+
+The most significant change is the removal of all features related to Earthly's commercial cloud offering.
+EarthBuild focuses on being a great, self-hosted build tool.
+
+Features related to the cloud-hosted earthly commercial offering were removed in the [final release of earthly
+`v0.8.16`](https://github.com/earthly/earthly/releases/tag/v0.8.16) and will never be present in EarthBuild
+releases.
+
+We will maintain compatibility while logging warnings for other, more invasive, changes for releases of
+EarthBuild on the `v0.8.x` minor version.
+
+We will publish a breaking change to these features in the first unique minor version for EarthBuild, `v0.9.x`.
+
+These changes include renaming of configuration variables from `EARTHLY_*` to `EARTHBUILD_*`, removal of Earthfile syntax related to cloud
+hosting like `PROJECT` and naming of built-in arguments like `ARG EARTHLY_GIT_PROJECT_NAME` to `ARG EARTHBUILD_GIT_PROJECT_NAME`.
+
+### Binary Name Change
+
+The command-line tool has been renamed from `earthly` to `earth`. You will need to update your scripts, CI configurations, and any local aliases.
+
+```diff
+- earthly +all
++ earth +all
+```
+
+In the `earthlybuild/actions-setup` github action, we've aliased `earthly` to `earth`, logging the deprecated
```

Comment:
In an ideal scenario, we could add a `migrate` command to the `earth` CLI to assist users in migrating. Also, it would be helpful to migrate deprecated stuff in the future.

My concern is that we don't have enough contributors to help us.